packages/ui/src/components/sha-badge.tsx

```diff
@@ -43,10 +43,10 @@ function Icon({ ...props }: IconProps) {
 
   return (
     <div
-      onClick={handleClick && handleClick}
       role="button"
       tabIndex={0}
-      className="bg-background hover:bg-muted/50 flex items-center border-l px-1.5 py-0.5"
+      onClick={handleClick && handleClick}
```

Comment:
This is a bit weird and we can simplify with the same result
```suggestion
      onClick={handleClick}
```